### PR TITLE
fix race condition for test case when multiple ttos are deployed

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
fix race condition for test case when multiple ttos are deployed

This commit patches the test, so it can try to check TTO conditions multiple times (every 2 seconds for 60 seconds), because it sometimes can take some time until reconciliation is completed

Signed-off-by: Karel Šimon <ksimon@redhat.com>

**Release note**:
```
NONE
```

### CODE REVIEW

File: tests/tekton-tasks_test.go [hunk 1]
```diff
@@ -283,22 +282,28 @@ var _ = Describe("Tekton-tasks", func() {
...
+						if condition.Type == conditionsv1.ConditionAvailable && condition.Status == v1.ConditionTrue {
+							return false
+						}
+						if condition.Type == conditionsv1.ConditionProgressing && condition.Status == v1.ConditionTrue {
+							return false
+						}
+						if condition.Type == conditionsv1.ConditionDegraded && condition.Status == v1.ConditionFalse {
+							return false
+						}
+
```
Comment (jcanocan) [inline tests/tekton-tasks_test.go:299]: nit: remove blank line between if's.

**Review by jcanocan — CHANGES_REQUESTED** [review]
Thanks for your PR @ksimon1. I just have a small comment.

File: tests/tekton-tasks_test.go (hunk 1 above)
Comment (ksimon1) [reply tests/tekton-tasks_test.go]: done

**Pull request merged**